docs/documentation/governance.md

```diff
@@ -3,7 +3,7 @@
 - Learn about the [**Aragon Governance Proposal process**](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-1.md)
 - Learn [**how the Aragon Assocation Board of Directors votes on proposals**](https://github.com/aragon/AGPs/blob/master/aa_board_review.md)
 - Learn and discuss [**how to submit an Aragon Governance Proposal**](https://forum.aragon.org/t/how-to-create-an-aragon-governance-proposal/374)
-- Check out [**all open Aragon Governance Proposals**](https://github.com/aragon/AGPs/pulls)
+- Check out [**all currently open Aragon Governance Proposals**](https://github.com/aragon/AGPs/pulls)
```

Comment:
@Smokyish  "currently" does not seem to add any value here... there is no such thing as a "past open" or "future open" proposal, it is either open or it's not.